src/pages/state/[state].js

```diff
@@ -148,102 +172,95 @@ export default function State({ data: { statesGeoData }, state }) {
-                <Row>
-                  <ul>
-                    {stateData.voicesForReform.map(({ note }, i) => (
-                      <li key={"vfr-notes-" + i}>
-                        <ReactMarkdown>{note}</ReactMarkdown>
-                      </li>
-                    ))}
-                  </ul>
-                </Row>
-              ) : null}
-              {stateData.videos?.length
-                ? stateData.videos.map(({ link, title, description }, i) => (
-                    <Row key={`statewide-videos-${i}`}>
-                      <ReactPlayer url={link} width={594} height={361} />
-                      <h4>{title}</h4>
-                      {description && (
-                        <ReactMarkdown className={styles.description}>
-                          {description}
-                        </ReactMarkdown>
-                      )}
-                    </Row>
-                  ))
-                : null}
+        )}
+      </Row>
+      {/* This data will be the bar charts
+        {inStateOpos
+          .filter(({ notes }) => notes?.length)
+          .map(({ name, notes }) => (
+            <Row key={name}>
+              <h4>{name}</h4>
+              <ul>
+                {notes?.map((note, i) => (
+                  <li key={`${name}-note-${i}`}>
+                    <ReactMarkdown>{note}</ReactMarkdown>
+                  </li>
+                ))}
+              </ul>
+            </Row>
+          ))} */}
+      {stateData.videos?.length || stateData.voicesForReform?.length ? (
+        <Row className={styles.voices}>
+          <Row>
+            <h3>Voices For Reform</h3>
+          </Row>
+          {stateData.voicesForReform?.length ? (
+            <Row>
+              <ul>
+                {stateData.voicesForReform.map(({ note }, i) => (
+                  <li key={"vfr-notes-" + i}>
+                    <ReactMarkdown>{note}</ReactMarkdown>
+                  </li>
+                ))}
+              </ul>
             </Row>
           ) : null}
-        </Col>
-      </Row>
+          {stateData.videos?.length
+            ? stateData.videos.map(({ link, title, description }, i) => (
+                <Row key={`statewide-videos-${i}`} className={styles.video}>
+                  <ReactPlayer url={link} width={594} height={361} />
+                  <h4>{title}</h4>
+                  {description && (
+                    <ReactMarkdown className={styles.description}>
+                      {description}
+                    </ReactMarkdown>
+                  )}
+                </Row>
+              ))
+            : null}
+        </Row>
+      ) : null}
+      {outOfStateOpos.length > 0 && (
+        <Row className={styles.serviceTable}>
+          <OpoTable
+            headings={headings}
+            inState={false}
+            opos={outOfStateOpos}
+            title="OPO Performance in Neighboring States"
+          />
+        </Row>
+      )}
     </Layout>
```

Comment:
out of scope for this PR, but this file is very long. consider stripping out some of the pre-processing that's done in the top of this component function into separate functions that live in util files or even just at the bottom of this file like the query below; would likely improve readability of the component. to that end, could be helpful to future devs to include short comments that quickly demarcate and define the various sections of this component JSX, even just a one liner before each top-level `Row` object, i.e. 
line 121:
```// Defines which OPOs serve this state, and links to more data about those OPOs```
line 130:
``` // State map data with formatted statistics ```
line 179: 
``` // Notes and news about state, or serving OPOs ```
... etc
Yes, this is figure-out-able by reading the code, but having to track depth of the various `Row` objects to be sure of the data differentiation and organization is a bit tricky. Alternative to inline comments would be pulling each of these sections out into separate named components for readability + shortening this file, not for re-usability so arguments could be made against this approach. but I personally would rather read through multiple short components each with a really clear, explanatory name than one long file. If it's gonna be one long file, I love comments!

But again -- not really within scope for this change, and so not a blocker, Just something to consider